src/components/CreateCredential.js

```diff
@@ -17,53 +20,116 @@ const CreateCredential = () => {
       }
     }
   `
-  );
+  )
 
   return (
+    <GetCredential className="getCredentialContainer" templateId={GATSBY_TEMPLATE_ID} productName={GATSBY_PRODUCT_NAME} >
 
-    <GetCredential credentialType="apiKey" className="getCredentialContainer" service="CCEmbedCompanionAPI">
+      <GetCredential.SignIn title="Get credentials" paragraph="Create unique credentials that you will use to call multiple APIs from your application." buttonText="Sign in to create credentials" />
 
-      <GetCredential.SignIn title="Get credentials" paragraph="Create unique credentials that you will use to call the Adobe Express Embed SDK from your application." buttonText="Sign in to create credentials" className="SignInClass" />
+      <GetCredential.Form title="Get credentials" paragraph="Create unique credentials that you will use to call multiple APIs from your application." className="formClass">
```

Comment:
Instead of multiple APIs, shouldn't this say `Adobe Express Embed SDK`? Since we only have one API in this case.